sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -96,16 +113,24 @@ export interface PartitionOwnership {
 }
 
 /**
- * The PartitionProcessorFactory is called by EPH whenever a new partition is about to be processed.
+ * A functional interface to create new instance(s) of `PartitionProcessor` when provided with a `PartitionContext` and `CheckpointManager`.
  */
 export interface PartitionProcessorFactory {
+  /**
+   * Factory method to create a new instance(s) of `PartitionProcessor` for a partition.
```

Comment:
Since we have the `for a partition` in the end of the sentence, we are dealing with a single instance.

```suggestion
   * Factory method to create a new instance of `PartitionProcessor` for a partition.
```